pandas/tests/arithmetic/test_numeric.py

```diff
@@ -332,6 +332,9 @@ def test_ser_divmod_zero(self, dtype1, any_real_dtype):
         right = pd.Series([0, 2]).astype(dtype2)
 
         expected = left // right, left % right
```

Comment:
in future PR can you give some comments on what you ar doing here (filling np.inf on the integer divide)

Reply:
sure